test/dotnet-test.Tests/CollectCodeCoverage.runsettings

```diff
@@ -1,25 +0,0 @@
-<RunSettings>
```

Comment:
Do we want to delete this?

Reply:
I don't even understand how this is deleted. Like, fundamentally. I made sure to select files from this directory to use the SDK's history. So this shouldn't have happened. Even if I undelete it now, I can't preserve the history of this file. I'd have to redo the entire PR for just this file.

Looks like this file was deleted from the Installer repo in 2018. I think the only reasonable solution is to just add it back in this PR and have the history for it be broken.

Reply:
Oh oh oh. I think I knew what happened. Wait. Let me check something...